input/fsh/SD_RadiotherapyPhasePrescription.fsh

```diff
@@ -2,23 +2,23 @@
 Profile: RadiotherapyPhasePrescription
```

Comment:
reasonCode and reasonReference are MS but not bound to anything.
Should reason Code be bound to a disease code in SNOMED/ICD-10?
Should reasonReference be bound to a Reference(Condition)?